rskj-core/src/main/java/co/rsk/rest/modules/HealthCheckModule.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2018 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package co.rsk.rest.modules;
+
+import co.rsk.rest.RestUtils;
+import io.netty.handler.codec.http.DefaultFullHttpResponse;
+import io.netty.handler.codec.http.HttpMethod;
+
+public class HealthCheckModule extends RestModule {
```

Comment:
I would separate the  Module logic from the Rest implementation, so if for any reason we move from rest to json-rpc  (for example) we only would have to change one part and not he hole module implementation.

Reply:
We already have other types of servers, and if we decide to move some modules between them, we would have to refactor plenty of things, since implementations between servers differs a lot.

As a side note, the whole idea of having a `REST` server is to have some clear separation between different modules or things you can do with the node. In this case, `Health check` feature is going to be accessed only via its `REST API`.